src/main/java/net/sacredlabyrinth/phaed/simpleclans/SimpleClans.java

```diff
@@ -256,13 +262,10 @@ public String getLang(@NotNull String key, @Nullable Player player) {
     }
 
     @Nullable
-    public static String optionalLang(@NotNull String key, @Nullable Player player, Object... arguments) {
-        Locale locale = getInstance().getSettingsManager().getLanguage();
-        if (player != null && instance.getSettingsManager().isLanguagePerPlayer()) {
-            Locale playerLocale = Helper.getLocale(player);
-            if (playerLocale != null) {
-                locale = playerLocale;
-            }
+    public static String optionalLang(@NotNull String key, @Nullable ClanPlayer clanPlayer, Object... arguments) {
+        Locale locale = instance.getSettingsManager().getLanguage();
+        if (clanPlayer != null && instance.getSettingsManager().isLanguagePerPlayer()) {
+            locale = Helper.getLocale(clanPlayer);

```

Comment:
Helper.getLocale() can be deleted with replacing on clanPlayer.getLocale()

Reply:
Sure